DEA_products/DEA_Fractional_Cover.ipynb

```diff
@@ -87,538 +87,6 @@
    "cell_type": "code",
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #1.    # List DEA Fractional Cover products available in DEA
</pre><p>The "with warnings.catch_warnings():" code doesn't seem to have actually stopped the warnings, so we might want to get rid of it to keep things simple. Hopefully there will be a better fix soon</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GeoscienceAustralia/dea-notebooks/pull/1402/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='9'/>